### ISSUE TITLE
Memory leak on log.setup

### ISSUE THREAD

**Issue by xandros6** [DaveWoodCom/XCGLogger#194]
Version 8.3 (8E162) . Swift 3.1
XCGLogger at "4.1.0"

```
class AppDelegate: UIResponder, UIApplicationDelegate, UNUserNotificationCenterDelegate {
...
let log = XCGLogger.default
...
func application(_ application: UIApplication, didFinishLaunchingWithOptions launchOptions: [UIApplicationLaunchOptionsKey: Any]?) -> Bool {
...
log.setup(level: .debug, showFunctionName: false, showThreadName: false, showLevel: true, showFileNames: true, showLineNumbers: true, writeToFile: logPath, fileLevel: .debug)
```

On last line app crash reporting 

> Message from debugger: Terminated due to memory issue

**Comment by sohayb** [comment]
This is the same issue as #193

**Comment by DaveWoodCom** [comment]
Thanks. Tracking this under #193. But your `XCGLogger at "4.1.0"` comment has me concerned. Since there is no 4.1.0 yet. #195 seems related to that as well.

**Comment by sohayb** [comment]
@xandros6 issue is fixed now. Can you please close this?

**Comment by DaveWoodCom** [comment]
Think this was an issue with the bad tags, which should now be fixed with the 5.0.1 release. But please re-open this if you continue to see issues.
